modules/local/gatk4/haplotypecaller/main.nf

```diff
@@ -8,7 +8,7 @@ process GATK4_HAPLOTYPECALLER {
         'quay.io/biocontainers/gatk4:4.2.3.0--hdfd78af_0' }"
 
     input:
-    tuple val(meta), path(cram), path(crai), path(interval)
+    tuple val(meta), path(cram), path(crai), path(intervals_bed)
```

Comment:
the gatk4 modules are in the process of being pulled from nf-core/modules. we should reflect any changes from there, we need double check with the upstream modules